<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendToDialog</class>
 <widget class="QDialog" name="SendToDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Destination Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Address Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="broadcastAllRadioButton">
        <property name="text">
         <string>Broadcast All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="broadcastRoutersRadioButton">
        <property name="text">
         <string>Broadcast Routers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="broadcastRxOnWhenIdleRadioButton">
        <property name="text">
         <string>Broadcast Rx On When Idle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="groupRadioButton">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="unicastRadioButton">
        <property name="text">
         <string>Unicast</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Endpoint</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="addressEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="inputMask">
        <string>\0\xHHHH; </string>
       </property>
       <property name="text">
        <string>0x0000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="endpointEdit">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>\0\xHH; </string>
         </property>
         <property name="text">
          <string>0x00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
